app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,38 @@
+<% content_for :title do "Login" end %>
```

Comment:
This is our login form and has been tweaked/changed from the generated devise view.